src/lib/reasoners/fun_sat.ml

```diff
@@ -1924,6 +1925,18 @@ module Make (Th : Theory.S) : Sat_solver_sig.S = struct
     Shostak.Combine.reinit_cache ();
     Uf.reinit_cache ()
 
+  let get_model env =
+    assert (Options.get_interpretation ());
+    if not @@ Options.get_dump_models () then
+      let env = compute_concrete_model env true in
+      Options.Time.unset_timeout ~is_gui:(Options.get_is_gui());
+      let prop_model = extract_prop_model ~complete_model:true env in
+      Th.output_concrete_model (Options.Output.get_fmt_mdl ()) ~prop_model
+        env.tbox;
+      terminated_normally := true
+    else
+      Printer.print_wrn "get-model commands are ignored in presence of \
+                         --model option."
```

Comment:
We should not ignore `get-model` here, see above.

Reply:
Actually, the command is not ignored but useless in presence of the flag `--model`. We can print nothing.

Reply:
What do you mean by "print nothing"? We should print the model.